repo.path

```diff
@@ -1,8 +1,5 @@
-# common
-ostis-web-platform/kb
-
 # components
-kb/
+knowledge-base/
 
 # specifications
```

Comment:
the fact that agents specification is in problem-solver requires to put all project in 'kb' storage inside docker
if someone has videos or large files in documentation or at other locations then everything is put in that storage
should it be an issue?

Reply:
For this we have repo.path, where we can specify multiple paths to the knowledge base folders

Reply:
we do have that
but line 23 in docker-compose.yml loads entire project inside kb volume
and this is just to add 2 files not from knowledge-base folder to volume that will be built later
I don't find this ok, that volume is used for knowledge base builds and it should contain only files that are essential for knowledge base build

Reply:
soo
why isn't this an issue?

Reply:
Fixed